i18n/fr/docusaurus-plugin-content-docs/version-23.04/resources/glossary.md

```diff
@@ -74,9 +74,11 @@ On appelle aussi “collecteur” le moteur de supervision présent dans un serv
 
 ## Connecteur de supervision
 
-L'expression "Connecteur de supervision" désigne un [plugin](#plugin) et son pack :
+Un connecteur de supervision est la brique logicielle qui permet à Centreon de communiquer avec les équipements supervisés.
```

Comment:
Il faudrait traduire pour refléter la dernière modif en EN (Connectors are the piece...)